bitnami/external-dns/templates/_helpers.tpl

```diff
@@ -128,6 +128,8 @@ Return true if a secret object should be created
     {{- true -}}
 {{- else if and (eq .Values.provider "ns1") .Values.ns1.apiKey (not .Values.ns1.secretName) -}}
     {{- true -}}
+{{- else if and (eq .Values.provider "civo") .Values.civo.apiToken -}}
+    {{- true -}}
```

Comment:
I think you missed adding the related logic for `secretName`:

```suggestion
{{- else if and (eq .Values.provider "civo") .Values.civo.apiToken (not .Values.civo.secretName) -}}
    {{- true -}}
```

Reply:
Indeed. Fixed!